coretex/_task/_local.py

```diff
@@ -101,6 +112,15 @@ def configure(self) -> None:
         self.add_argument("--name", nargs = "?", type = str, default = None)
         self.add_argument("--description", nargs = "?", type = str, default = None)
 
+        for parameter in self.parameters:
+            if parameter.dataType in [ParameterType.dataset, ParameterType.enum, ParameterType.enumList, ParameterType.imuVectors]:
+                self.add_argument(f"--{parameter.name}", nargs = "?", type = parameter.overrideValue, default = None)
+            elif parameter.dataType.value.startswith("list"):
+                parameter = cast(BaseListParameter,  parameter)
```

Comment:
Instead of checking if `dataType` value starts with a list string and using `cast` a simple check can be performed via `isinstance(object, type)` and in a case where this is true python linter will automatically cast that object into the specified type and treat it as such.